bin/package-production.js

```diff
@@ -0,0 +1,129 @@
+var AdmZip = require('adm-zip');
+var colors = require('colors');
+var cp = require('child_process');
+var fs = require('fs');
+var join = require('path').join;
+var os = require('os');
+var path = require('path');
+var Promise = require("bluebird");
+var resolve = require('path').resolve;
+var rimraf = require('rimraf');
+
+// Get Lambda Parent Folder
+var src = resolve(__dirname, '../src/');
+
+// Get Distributions Parent Folder
+var dist = resolve(__dirname, '../dist/');
+
+// Create Zip Handler
+var zip = new AdmZip();
+
+// Determine npm binary based on OS
+var cmd = os.platform().startsWith('win') ? 'npm.cmd' : 'npm';
+
+/**
+ * Check that Lambas have required assets
+ * @param  {String} lambdaPath Absolute Path to Lambda Folder
+ * @param  {String} name Name of Lambda Function
+ */
+function checkLambda (lambdaPath, name) {
+  return new Promise(function(resolve, reject) {
+    if (!fs.existsSync(join(lambdaPath, 'package.json'))) {
+      reject(`× ERROR: '${name}' missing package.json`);
+    } else {
+      resolve(`\n Packaging '${name}' lambda ... \n`);
+    }
```

Comment:
@manifestinteractive  this is super small, but it's a thing of mine - I like to return whenever possible to avoid conditionals / bracketing - so `return reject(...); resolve(...)`. just a style thing